time/slots/slottime.go

```diff
@@ -163,6 +163,12 @@ func ToTime(genesisTimeSec uint64, slot primitives.Slot) (time.Time, error) {
 	return time.Unix(int64(sTime), 0), nil // lint:ignore uintcast -- A timestamp will not exceed int64 in your lifetime.
 }
 
+// SlotStart computes the timestamp where the given slot begins, relative to the genesis timestamp.
+func BeginsAt(slot primitives.Slot, genesis time.Time) time.Time {
```

Comment:
look mom, no `error`!

Reply:
This one needs  a test, it's exported and is critical for blob verification.

Reply:
Added a couple test cases. They are pretty trivial, as I can't think of a way to break this function, so please let me know if you think of other things to test.

Reply:
what is the difference between `BeginsAt` and `ToTime` ? They look like they are doing the same thing